### DISCUSSION TITLE
ddns "stop ddns"clicks are unresponsive

### DISCUSSION THREAD

**Discussion by science2468** [immortalwrt/immortalwrt#1038] [Q&A]
### Describe the bug

"stop ddns", "restart ddns", "Update DDns Services List" all click unresponsively

### ImmortalWrt version

23.05.0-rc3 (r26804-a3639e76d6)

### ImmortalWrt target/subtarget

x86/64

### Device

ASUSTeK COMPUTER INC. F1A55-M LX3 PLUS R2.0

### Image kind

Official downloaded image

### Additional info

![image](https://github.com/immortalwrt/immortalwrt/assets/102643084/3fe5a00d-3baa-42f2-93a9-a19305e7009b)

### Terms

- [X] I am reporting an issue for ImmortalWrt, not an unsupported fork.

**Comment by 1715173329** [comment]
You don't even enable them, why do you think they will work?
If you don't know how to use it, just install ddns-go instead.

**Comment by science2468** [comment]
> You don't even enable them, why do you think they will work? If you don't know how to use it, just install ddns-go instead.

ddns is working, but clicking "stop ddns", "restart ddns", "Update DDns Services List" does not respond. On the system-startup can start and shut down ddns.

**Comment by 1715173329** [comment]
I can only see log says "Service section disabled".
Paste your full ddns configuration here.

**Comment by science2468** [comment]
> I can only see log says "Service section disabled". Paste your full ddns configuration here.

![image](https://github.com/immortalwrt/immortalwrt/assets/102643084/5af21fc9-b716-4603-89a3-905cf4b26e00)
